### ISSUE TITLE
Do you still need commit access?

### ISSUE THREAD

**Issue by tstellar** [llvm/llvm-project#81548]
### TLDR: If you want to retain your commit access, please comment on this issue.  Otherwise, you can unsubscribe from this issue and ignore it.

@mprobst, @vmiklos, @gislan, @tschwinge, @Theodor, @tinti, @agners, @ianlevesque, @steveire, @MadCoder

LLVM has a policy of downgrading write access to its repositories for accounts with long term inactivity. This is done because inactive accounts with high levels of access tend to be at increased risk of compromise and this is one tactic that the project employs to guard itself from malicious actors.

Our project policy is to ping anyone with less than five 'interactions' with the repositories over a 12 month period to see if they still need commit access.  An 'interaction' and be any one of:

* Pushing a commit.
* Merging a pull request (either their own or someone else’s).
* Commenting on a PR.

If you want to retain your commit access, please post a comment on this issue.  If you do not want to keep your commit access, you can just ignore this issue.  If you have not responded in 6 weeks, then you will move moved from the 'write' role within the project to the 'triage' role.  The 'triage' role is still a priviledged role and will allow you to do the following:

* Review Pull Requests.
* Comment on issues.
* Apply/dismiss labels.
* Close, reopen, and assign all issues and pull requests.
* Apply milestones.
* Mark duplicate issues and pull requests.
* Request pull request reviews.
* Hide anyone’s comments.

**Comment by ianlevesque** [comment]
Yes

**Comment by vmiklos** [comment]
> If you want to retain your commit access, please post a comment on this issue.

Yes, would like to keep it if possible. I hope to get back to contributing to clang-tidy again, in the future.

**Comment by tschwinge** [comment]
Yes, planning to get more active in LLVM eventually.

**Comment by agners** [comment]
No, not at the moment anyways. Please feel free to remove my commit access. Thanks! :+1:

**Comment by tinti** [comment]
No. Please remove my commit access.

**Comment by gislan** [comment]
Yes, I'd like to keep it if possible.

**Comment by tstellar** [comment]
@mprobst @Theodor @steveire @MadCoder Reminder, if you want to retain your commit access, please comment on this ticket in the next week.

**Comment by tstellar** [comment]
@tinti @agners @mprobst @Theodor @steveire @MadCoder added to the LLVM Triagers team.  If you need write access again, please follow these instructions:  https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access
